core/src/test/scala/org/apache/spark/util/collection/ExternalAppendOnlyMapSuite.scala

```diff
@@ -253,7 +253,7 @@ class ExternalAppendOnlyMapSuite extends SparkFunSuite
   private def testSimpleSpilling(codec: Option[String] = None, encrypt: Boolean = false): Unit = {
     val size = 1000
     val conf = createSparkConf(loadDefaults = true, codec)  // Load defaults for Spark home
-    conf.set("spark.shuffle.spill.numElementsForceSpillThreshold", (size / 4).toString)
+    conf.set(SHUFFLE_SPILL_NUM_ELEMENTS_FORCE_SPILL_THRESHOLD.key, (size / 4).toString)
```

Comment:
One broad question about most of these changes: can this not be:
`conf.set(SHUFFLE_SPILL_NUM_ELEMENTS_FORCE_SPILL_THRESHOLD, size / 4)`?
In Scala at least? I thought the idea of this typed API was that you could pass a typed ConfigEntry and matching typed argument.

Reply:
It looks better. I'll try to update it. Thanks